packages/types.ts

```diff
@@ -48,6 +48,17 @@ export interface ToastIcons {
   loading?: Component
 }
 
+export type ToastAction = {
+  label: string | Component
+  onClick: (
+    event: MouseEvent,
+    toast: {
+      deleteToast: (delay?: number) => void
+    }
+  ) => void
+  classes?: string
```

Comment:
Could you elaborate on the purpose of the `classes` field?

Reply:
If anybody wants to append custom style to each action's button